From 409b9a9fe4650272d471ee9d924d30b508362779 Mon Sep 17 00:00:00 2001
From: zhangqing <zhangqing@rock-chips.com>
Date: Mon, 13 Apr 2015 04:27:48 -0700
Subject: [PATCH] rk3368: clk: powerdomain: enable powerdomain func.

test:Passed the special test.Test machine 15 for P9 and test 12H.
	Have the test report form Quality department.

Signed-off-by: zhangqing <zhangqing@rock-chips.com>
---
 arch/arm64/boot/dts/rk3368.dtsi | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/arch/arm64/boot/dts/rk3368.dtsi b/arch/arm64/boot/dts/rk3368.dtsi
index f9e36c5eaab8..7995a7007fdc 100755
--- a/arch/arm64/boot/dts/rk3368.dtsi
+++ b/arch/arm64/boot/dts/rk3368.dtsi
@@ -617,11 +617,6 @@
 	rockchip_clocks_enable: clocks-enable {
 		compatible = "rockchip,clocks-enable";
 		clocks =
-		<&pd_vio>,
-               <&pd_video>,
-               <&pd_gpu_0>,
-               <&pd_gpu_1>,
-
 			/*PLL*/
 			<&clk_apllb>,
 			<&clk_aplll>,
-- 
2.35.3

